tests/telemetry/test_telemetry.py

```diff
@@ -73,9 +73,10 @@ def assert_equal(actual, expected, message):
     pytest_assert(actual == expected, "{0}. Expected {1} vs actual {2}".format(message, expected, actual))
 
 @pytest.fixture(scope="module", autouse=True)
-def verify_telemetry_dockerimage(duthosts, rand_one_dut_hostname):
+def verify_telemetry_dockerimage(duthosts, rand_one_dut_hostname, ptfhost):
     """If telemetry docker is available in image then return true
     """
+    global GNXI_PATH
```

Comment:
Can you make it a fixture instead of global variable?

Reply:
Done